fhireval/test_suite/search/test_paging.py

```diff
@@ -1,15 +1,34 @@
 import pytest
 import pdb
 
-from fhir_walk.model.patient import Patient
+from fhireval.data import prep_server
 
-test_id = f"{'2.2.1.9.2':<10} - Paging"
+test_id = f"{'2.10.1.9.2':<10} - Paging"
 
 test_weight = 8
 
-def test_paging(host):
-    assert 0 == 1, "TODO - Write Test"
+def test_paging(host, prep_server):
+    # We know there are more than 5 patients, so let's just go with that
+    qry_response = host.get(f"Patient", recurse=False, rec_count=5, raw_result=True)
+    assert qry_response['status_code'] == 200
+    assert qry_response['response']['resourceType'] == 'Bundle', "Sanity check that we got a bundle like expected"
+    assert qry_response['response']['total'] > 5, "Make sure we got more records than our max page size"
+    assert len(qry_response['response']['entry']) == 5, "Did we get only 5 records"
+    assert qry_response['response']['link'][1]['relation'] == 'next', "Do we have a link for the next page?"
```

Comment:
I know in `"link"`, `[0]` is `"self"` while `[1]` is `"next"`, but assuming this behavior always works may fail in the future specification updates. So, why don't we loop over the `"link"` list?

Reply:
Done.